linux_os/guide/system/auditing/auditd_configure_rules/file_permissions_var_log_audit/ansible/shared.yml

```diff
@@ -0,0 +1,63 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_sle
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+- name: Get audit log files
+  command: grep -iw ^log_file /etc/audit/auditd.conf
+  failed_when: false
+  register: log_file_exists
+
+- name: Parse log file line
+  command: awk -F '=' '/^log_file/ {print $2}' /etc/audit/auditd.conf
+  register: log_file_line
+  when: (log_file_exists.stdout | length > 0)
+
+- name: Set default log_file if not set
+  set_fact:
+    log_file: "/var/log/audit/audit.log"
+  when: (log_file_exists is undefined) or (log_file_exists.stdout | length == 0)
+
+- name: Set default log_file if not set
+  set_fact:
+    log_file: "{{ log_file_line.stdout | trim }}"
+  when: (log_file_line.stdout | length > 0)
```

Comment:
What if log_file_line is undefined? It can easily happen. If log_file_exists is false then the task "Parse log file line" isn't executed at all, and then this task will error.

Reply:
Thx for the tip :bow: should be handled in [b5b34f9](https://github.com/ComplianceAsCode/content/pull/9790/commits/b5b34f9f9c3727f3b9c25535a75d1da7245310ba)